notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>Most of the other steps call this a "tutorial" instead of example. I'd stay consistent or drop it completely.</p><p> </p><p>This tutorial uses a pretrained image model <a href="https://pytorch.org/vision/master/models/generated/torchvision.models.resnet18.html" target="_blank">resnet18</a> from the PyTorch TorchVision.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>